screens/ChangePassword.screen.jsx

```diff
@@ -6,83 +6,62 @@ import colors from 'themes/colors';
+      withLogo={false}
+    >
+      <View style={styles.imageContainer}>
+        <FImage
+          imagePath={images.CHANGE_PASSWORD()}
+          width={sizes.WIDTH_120}
+          height={sizes.HEIGHT_120}
+        />
+      </View>
+      <FInput
+        iconPlacement={placements.LEFT}
+        type={inputTypes.PASSWORD}
+        icon={icons.LOCK_CLOSED_OUTLINE}
+        placeholder={locales.PASS_OLD_PASSWORD}
+        marginBottom={sizes.MARGIN_30}
       />
-    </View>
-    <FHeading
-      title={locales.PASS_OLD_PASSWORD}
-      align={placements.LEFT}
-      size={fonts.HEADING_NORMAL}
-      weight={fonts.HEADING_WEIGHT_SEMIBOLD}
-      color={colors.DARK_GRAY}
-      marginBottom={sizes.MARGIN_20}
-    />
-    <FInput
-      iconPlacement={placements.LEFT}
-      type={inputTypes.PASSWORD}
-      icon={icons.LOCK_CLOSED_OUTLINE}
-      placeholder={locales.OLD_PASSWORD}
-    />
-    <FHeading
-      title={locales.PASS_NEW_PASSWORD}
-      align={placements.LEFT}
-      size={fonts.HEADING_NORMAL}
-      weight={fonts.HEADING_WEIGHT_SEMIBOLD}
-      color={colors.DARK_GRAY}
-      marginBottom={sizes.MARGIN_20}
-    />
-    <FInput
-      iconPlacement={placements.LEFT}
-      type={inputTypes.PASSWORD}
-      icon={icons.LOCK_CLOSED_OUTLINE}
-      placeholder={locales.NEW_PASSWORD}
-    />
-    <FHeading
-      title={locales.REPEAT_NEW_PASSWORD}
-      align={placements.LEFT}
-      size={fonts.HEADING_NORMAL}
-      weight={fonts.HEADING_WEIGHT_SEMIBOLD}
-      color={colors.DARK_GRAY}
-      marginBottom={sizes.MARGIN_20}
-    />
-    <FInput
-      iconPlacement={placements.LEFT}
-      type={inputTypes.PASSWORD}
-      icon={icons.LOCK_CLOSED_OUTLINE}
-      placeholder={locales.NEW_PASSWORD}
-    />
-    <View style={styles.buttonContainer}>
-      <FButton
-        title={locales.CHANGE_PASSWORD}
-        type={buttonTypes.TEXT_BUTTON}
-        backgroundColor={colors.DARK_PRIMARY}
-        color={colors.WHITE}
-        titleWeight={fonts.HEADING_WEIGHT_BOLD}
-        titleSize={fonts.HEADING_MEDIUM}
+      <FInput
+        iconPlacement={placements.LEFT}
+        type={inputTypes.PASSWORD}
+        icon={icons.LOCK_CLOSED_OUTLINE}
+        placeholder={locales.PASS_NEW_PASSWORD}
       />
-    </View>
-  </FDefaultLayout>
+      <FInput
+        iconPlacement={placements.LEFT}
+        type={inputTypes.PASSWORD}
+        icon={icons.LOCK_CLOSED_OUTLINE}
+        placeholder={locales.REPEAT_NEW_PASSWORD}
+      />
+      <View style={styles.buttonContainer}>
+        <FButton
+          title={locales.CHANGE_PASSWORD}
+          type={buttonTypes.TEXT_BUTTON}
+          backgroundColor={colors.DARK_PRIMARY}
+          color={colors.WHITE}
+          titleWeight={fonts.HEADING_WEIGHT_BOLD}
+          titleSize={fonts.HEADING_MEDIUM}
+        />
+      </View>
+    </FDefaultLayout>
+  </FKeyboardWrapper>
 );
 
-const styles = {
+const styles = StyleSheet.create({
   imageContainer: {
     width: sizes.WIDTH_FULL,
```

Comment:
Zamiast marginVertical = 30 daj żeby tylko ios miał ten margin (uzależnij od Platform). Dlatego że ios ma SafeArea, komponent, który dba o paddinig, a Android tego nie ma, dlatego jego layout ma ustawione paddingi na 30. Z tego względu trzeba uważać na to żeby nie dawać dodatkowych marginesów pierwszym komponentom w layout